### ISSUE TITLE
Airport lounges

### ISSUE THREAD

**Issue by 1ec5** [osmlab/name-suggestion-index#3541]
Airline-owned airport lounge brands should be listed in the index. For example:

* [Admirals Club](https://www.wikidata.org/wiki/Q84915954) (American Airlines)
* [Sky Club](https://www.wikidata.org/wiki/Q84914142) (Delta Airlines)
* [United Club](https://www.wikidata.org/wiki/Q7887610) (United Airlines)

There isn’t an established tag for airport lounges yet, just [this proposal](https://wiki.openstreetmap.org/wiki/Proposed_features/Lounges) for [`amenity=lounge`](https://taginfo.openstreetmap.org/tags/amenity=lounge), which has only been used a few times for airport lounges.

**Comment by andrewharvey** [comment]
In practice many are currently tagged as amenity=bar.

**Comment by bhousel** [comment]
This is a good idea but I wonder whether this should happen after we support `operator`

**Comment by 1ec5** [comment]
> In practice many are currently tagged as amenity=bar.

Yeah, I think it isn’t quite wrong to tag airport lounges as bars, but it’s also a bit of a stretch. They’re basically social clubs (for some definition of social) that happen to contain a bar or brunch buffet. I’ve mapped [piano lounges](https://en.wikipedia.org/wiki/Piano_bar) [as bars](https://www.openstreetmap.org/node/7171357711), so that makes me feel a little better about tagging airport lounges the same way.

> This is a good idea but I wonder whether this should happen after we support `operator`

I think support for `operator` would open up more possibilities for similarly-named facilities that aren’t branded per se, such as in train stations or some airports in Europe. But these lounges operated by U.S. airlines are commercially branded just like any restaurant chain.

**Comment by mds08011** [comment]
Lounges could definitely use some standardization: even my own tagging has been all over the place. 

I think I typically map them as a cafe with bar=yes and access restrictions.

Some of the larger non-airline brands such as Escape Lounges are also good candidates.

**Comment by tas50** [comment]
It would be really nice to get this standardized and added. There are easily dozens of different airline brands operating private lounges and those should really all end up in the index.

**Comment by TheAdventurer64** [comment]
And the preset still isn't available. Oof.

**Comment by bhousel** [comment]
> This is a good idea but I wonder whether this should happen after we support `operator`

Bumping this issue - we support an `operators` tree now.. Is there a settled tagging for airport lounges?

**Comment by TheAdventurer64** [comment]
At the moment, no.

**Comment by okainov** [comment]
>Bumping this issue - we support an operators tree now.. Is there a settled tagging for airport lounges?

It would be great to at least have `amenity=lounge` in general as something which is popping up as a hint when typing...
